src/lib.rs

```diff
@@ -74,18 +74,23 @@ use core::ptr::NonNull;
 use crate::convert::{FromWasmAbi, TryFromJsValue, WasmRet, WasmSlice};
 
 const _: () = {
-    /// The Wasm linker may or may not insert a call to this function from the beginning
-    /// of every function that your module exports. Specifically, it regards a module as
-    /// having "command-style linkage" if:
+    /// Used to get `__wasm_call_ctors` and other implicitly added functions that need to skip interpret.
```

Comment:
Maybe something like:

```suggestion
    ///  Dummy empty function provided in order to detect linker-injected functions like `__wasm_call_ctors` and others that must be skipped by the interpreter.
```

Reply:
thanks